### PR TITLE
fix make bundle

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR:
1. adds support to make yq part of make tools.
2. change the directory from config/manager to config/manifests while setting the image controller.
3. move the bundle.Dockerfile to bundles dir post generating bundle.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -269,16 +286,17 @@ endif
 .PHONY: bundle
 bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 	$(OPERATOR_SDK) generate kustomize manifests --apis-dir=./pkg/api --verbose
-	cd config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMG)
+	cd config/manifests && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMG)
 	$(KUSTOMIZE) build config/manifests | tee tmp/pre-bundle.yaml  |  \
 		$(OPERATOR_SDK) generate bundle --version $(VERSION) $(BUNDLE_GEN_FLAGS)
 	sed < bundle/manifests/kepler-operator.clusterserviceversion.yaml "s|containerImage.*|containerImage: $(OPERATOR_IMG)|g" \
```
Comment (sthaha) [inline Makefile:292]: is this needed? doesn't the `set image controller` above not work?

File: config/manifests/kustomization.yaml [hunk 2]
```diff
@@ -1,27 +1,15 @@
...
-#    namespace: system
-#  patch: |-
-#    # Remove the manager container's "cert" volumeMount, since OLM will create and mount a set of certs.
-#    # Update the indices in this path if adding or removing containers/volumeMounts in the manager's Deployment.
-#    - op: remove
-#      path: /spec/template/spec/containers/1/volumeMounts/0
-#    # Remove the "cert" volume, since OLM will create and mount a set of certs.
-#    # Update the indices in this path if adding or removing volumes in the manager's Deployment.
-#    - op: remove
-#      path: /spec/template/spec/volumes/0
```
Comment (sthaha) [inline config/manifests/kustomization.yaml:27]: Could you please maintain this? We will need webhooks soon to disallow creation of kepler instances other than kepler.

**Review by sthaha — CHANGES_REQUESTED** [review]
This commit does 3 things. We should break that in to 3 commits. 
1. patch that installs yq
2. fix the bundle
3. remove webhook part that is commented. 

Why is this important? 
lets say we want to revert the `fix bundle part` or `remove webhook` part, `git revert <commit>` is all that is needed. You can make 3 different commits from the same file using `git add -p` to select parts that form part of the commit. 


Do note the main reason for holding this PR is to make sure that we need / do not need the `sed`

**Review by vprashar2929 — COMMENTED** [review]
LGTM

**Review by sthaha — APPROVED** [review]

**Pull request merged**